pom.xml

```diff
@@ -122,6 +122,13 @@
 				<artifactId>wiremock-jre8</artifactId>
 				<version>${wiremock.version}</version>
 			</dependency>
+			<dependency>
```

Comment:
@robertmcnees can you take a look at this and make sure it aligns with the work you did with TestContainers?

Reply:
Is there a reason to import testcontainers twice?  The pom pulls in 1.15.1 already.  Could we change `${testcontainers.version}` instead?

Reply:
Oh sorry, missed that. I will update the property.